cfme/tests/physical_infrastructure/test_physical_rack_details.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
+def physical_rack(appliance, setup_provider_modscope):
+    # Get and return the first physical rack
+    physical_racks = appliance.collections.physical_racks.all()
```

Comment:
Required:
This test specific to `Lenovo` provider so better to used collection filtering
```
physical_racks = appliance.collections.physical_racks.filter({"provider": provider}).all()
```